.travis.yml

```diff
@@ -6,12 +6,29 @@ env:
 before_install:
 before_script:
 - npm install
-- npm install --save-dev chai karma karma-chai karma-chrome-launcher karma-coverage karma-firefox-launcher karma-html2js-preprocessor mocha karma-mocha karma-phantomjs-launcher karma-browserify babel-plugin-espower babelify karma-sauce-launcher
+- npm install --save-dev chai karma karma-chai karma-chrome-launcher karma-coverage
+  karma-firefox-launcher karma-html2js-preprocessor mocha karma-mocha karma-phantomjs-launcher
+  karma-browserify babel-plugin-espower babelify karma-sauce-launcher
 script:
 - npm test
+- VER_DEPL=$(npm show opentok-annotation version)
```

Comment:
@albertoacn  Need to [build](https://github.com/opentok/accelerator-annotation-js/blob/master/package.json#L14) the lib, and aren't you missing the check if this is a PR like this https://github.com/opentok/accelerator-core-js/blob/master/.travis.yml#L18 ?